app/src/main/java/ar/com/wolox/android/training/model/NewsItem.kt

```diff
@@ -0,0 +1,8 @@
+package ar.com.wolox.android.training.model
+
+import java.util.Date
+
+class NewsItem(val user: String, val message: String, val icon: String) {
```

Comment:
Use [`data class`](https://kotlinlang.org/docs/reference/data-classes.html) for this.